docs/src/hugo/content/index.md

```diff
@@ -80,7 +81,7 @@ Nelson does not support ad-hoc deployments. This is explicit and by-design. With
 
 Nelson leverages secure introduction. This means that the container does not hold any secrets until the very moment it is launched, and that the credentials given to the container dictate the access permissions the container has. **You cannot interact with a system or resource that you did not tell Nelson about**, and we'll cover how you inform Nelson about your dependencies in the next section.
 
-Practically speaking, credentials are sourced from [Vault](https://www.vaultproject.io/), and then mounted to a tempfs attached to the container. You can only source credentials from Vault, for which you have a valid Vault Policy. As it would turn out, these policies are dynamically generated by Nelson, and provisioned for you automatically. The credentials themselves should either be provisioned or configured by a member of staff with Vault access, and this should be done ahead of time, before you try to deploy your application with Nelson.
+Practically speaking, credentials are sourced from [Vault](https://www.vaultproject.io/) (or a KMS of your choosing), and then mounted to a `tmpfs` attached to the container. You can only source credentials from Vault, for which you have a valid Vault Policy. As it would turn out, these policies are dynamically generated by Nelson, and provisioned for you automatically. The credentials themselves should either be provisioned or configured by a member of staff with Vault access, and this should be done ahead of time, before you try to deploy your application with Nelson.
```

Comment:
Probably drop the "As it would turn out" from the sentence about dynamically generated policies